ffcx/codegeneration/ufcx.h

```diff
@@ -64,11 +64,13 @@ extern "C"
     ufcx_basix_element = 0,
     ufcx_mixed_element = 1,
     ufcx_quadrature_element = 2,
-    ufcx_custom_element = 3
+    ufcx_custom_element = 3,
```

Comment:
What is the difference between custom element and basix custom element?

Reply:
Good question...

Reply:
I've just checked and ufcx_custom_element isn't used anywhere, so I've removed it.

(As it was there before this PR, I wrongly assumed it was being used for something else)